docs/semgrep-supply-chain/getting-started.md

```diff
@@ -114,11 +114,17 @@ Additionally, you can view your results in Semgrep Cloud Platform (SCP). SCP dis
 
 ## Scan a monorepo's dependencies
 
-Semgrep Supply Chain supports the scanning of monorepos. It treates each of the monorepo's subdirectories as an individual repository. Findings are then grouped by the repositories based on the [lockfile](/semgrep-supply-chain/glossary/#lockfile) or manifest file present in the monorepo.
+Semgrep Supply Chain supports the scanning of monorepos. As outlined in [Project directory structure](#project-directory-structure), findings are grouped by directory based on the [lockfiles](/semgrep-supply-chain/glossary/#lockfile) or manifest files present in the monorepo.
 
 ## Block pull or merge requests
 
-Though Semgrep Supply Chain **v0.122.0** and earlier automatically blocked pull/merge requests if it discovered reachable findings in the code, it no longer does this. You can, however, configure your version control system to prevent merging if Semgrep Supply Chain identifies reachable findings.
+Semgrep Supply Chain versions **v0.122.0** and earlier automatically blocked pull/merge requests if it discovered reachable findings in the code, but later versions do not do this. You can, however, configure Semgrep Supply Chain to block on pull request scans that detect reachable findings with high or critical severity.
```

Comment:
Slight subtlety: we will block on any reachable code finding with these severities, but we will only block on a lockfile-only "always reachable" finding if it is high/critical and it's a _direct_ dependency. Always reachable findings on transitive dependencies won't block, because we thought it would be too noisy.

Reply:
It depends whether we consider it worth it to bring up the "directly importing a transitive dependency" edge case here. The blocking situation is this:

 |  | Reachable in code      | Always reachable in lockfile |
| ----------- | ----------- | ----------- |
| Direct dep      | Block       | Block |
| Transitive dep  | Block*        | No Block |

\* Only happens in edge case that should be avoided

Feels hard to keep it snappy 😅

Reply:
Yep - I'm not sure we need to cover the weird edge case in the glossary, but good point that we might want to consider covering it elsewhere - it might be a good fit for the KB.